### PR TITLE
Add query params filter to tesla middleware

### PR DESCRIPTION
**Pull request by moperacz**

### CODE REVIEW

File: test/tesla/middleware/meta_logger_test.exs [hunk 1]
```diff
@@ -8,9 +8,9 @@ defmodule Tesla.Middleware.MetaLoggerTest do
   defmodule FakeClient do
     use Tesla
 
-    plug(Subject, filter_headers: ["authorization"], log_level: :debug, log_tag: Subject)
+    plug Subject, filter_headers: ["authorization"], log_level: :debug, log_tag: Subject
```
Comment (zekus) [inline test/tesla/middleware/meta_logger_test.exs:11]: can you also test it at plug level?

Comment (moperacz) [reply test/tesla/middleware/meta_logger_test.exs:11]: yep :)

Comment (moperacz) [reply test/tesla/middleware/meta_logger_test.exs]: done

**Comment by moperacz** [comment]
@zekus did u maybe had some time to take a look?

**Review by zekus — APPROVED** [review]

**Review by nvieirafelipe — APPROVED** [review]

**Pull request merged**